sns.tf

```diff
@@ -29,7 +29,7 @@ POLICY
 
 resource "aws_sns_topic" "apiary_data_events" {
   count = "${ var.enable_data_events == "" ? 0 : length(var.apiary_managed_schemas) }"
-  name  = "${local.instance_alias}-${local.apiary_managed_schemas[count.index]}-data-events"
+  name  = "${local.instance_alias}-${local.apiary_managed_schema_names_original[count.index]}-data-events"
```

Comment:
This should be local.apiary_managed_schema_names_replaced as we use `-` to create AWS resource instead of `_`